src/WebJobs.Script.WebHost/Security/KeyManagement/SecretManager.cs

```diff
@@ -329,11 +342,42 @@ public async Task<bool> DeleteSecretAsync(string secretName, string keyScope, Sc
             }
         }
 
+        private ulong GetKeySeed(ScriptSecretsType secretsType, string keyScope)
+        {
+            if (secretsType == ScriptSecretsType.Function)
+            {
+                return SecretGenerator.FunctionKeySeed;
+            }
+            else if (secretsType == ScriptSecretsType.Host)
+            {
+                if (string.Equals(keyScope, HostKeyScopes.FunctionKeys, StringComparison.OrdinalIgnoreCase))
+                {
+                    return SecretGenerator.FunctionKeySeed;
+                }
+                else if (string.Equals(keyScope, HostKeyScopes.FunctionKeys, StringComparison.OrdinalIgnoreCase))
+                {
+                    return SecretGenerator.FunctionKeySeed;
```

Comment:
I am probably missing something simple, but it looks like both of these ifs check for the same thing and the branches both do the same thing

Reply:
Nice catch! My integration test didn't catch this because it only exercised Function keys, not system keys. I've added a unit test GetKeySeed_ReturnsExpectedResult.